src/client/app/components/LoginComponent.jsx

```diff
@@ -51,7 +51,15 @@ export default class LoginComponent extends React.Component {
 			localStorage.setItem('token', response.data.token);
 			browserHistory.push('/admin');
 		})
-		.catch(console.error);
+		.catch(() => {
+			this.inputEmail.focus();
+			this.props.sendNotification({
+				message: 'Invalid email/password combination',
+				level: 'error',
+				position: 'tr',
+				autoDismiss: 3
+			});
+		});
```

Comment:
Will this run on any error, no matter the status? It should be limited to whatever status code indicates an invalid password. (I.e., it should not run if the API returns 5xx).